CDCgov/NEDSS-Modernization
# Cnft1 1583 1612

https://cdc-nbs.atlassian.net/browse/CNFT1-1583
In advanced search page the patient search and event search tabs for visual consistency the highlight should not lead to the text being lifted, but instead the text should remain consistent across the line
Increasing transparent border in Advanced Search to ensure the switching of tabs does not cause a movement of the rest of the elements, that is because by increasing the transparent border that was previously 5px to 7px it ensures that when you click between patient search and event search that no visible movement is created.

Aligning Data Entry - Fields should be horizontally aligned -> https://cdc-nbs.atlassian.net/browse/CNFT1-1612

Steps to verify
1 .Login
2. Navigate to advanced search
3. Click repeatedly between Patient search and Event search tabs
4. Observe no movement with the change of the two tabs
